### PR TITLE
More portability improvements

### PR DESCRIPTION
**Pull request by jamc-code**
Hi! While creating the python script for country -> country code conversion, I ended up finding ways to improve portability.
Mainly, this is done with a `grep` call instead of the `cuts` we had previously. I also added a var called "VPN_PROVIDER" to allow users to clearly set that from the get go, and we can also use it in the `sed` script you mentioned to configure the main script.

Let me know what you think and have a great day!!
    - Julia

### CODE REVIEW

File: vpn_module.sh [hunk 1]
```diff
@@ -37,10 +37,11 @@ VPN_RELAY_SET_LOCATION="mullvad relay set location"
...
 # The first command cuts the status, which is compared to keywords below.
-# TODO: Add community submissions for other VPNs to make this section robust!
-VPN_STATUS="$($VPN_GET_STATUS | cut -d' ' -f3)"	# returns Connected/Connecting/<other>
-CONNECTED=Connected
-CONNECTING=Connecting
+# Note from Julia: this should cover most VPNs, if it's missing something let me know
+VPN_STATUS="$($VPN_GET_STATUS | grep -Eio 'connected|connecting|disconnected' \
+	| tr '[:upper:]' '[:lower:]')"
+CONNECTED=connected
+CONNECTING=connecting
```
Comment (jamc-code) [inline vpn_module.sh:45]: run vpn status command, look for keywords and translate it to lower case. the translation is so vpn providers using {CONNECTED,Connected,connected} will all register the same

File: country_codes.py [hunk 2]
```diff
@@ -49,5 +52,4 @@
 if country in countries:
     print(countries[country])
 else:
-    print("Not in country dict.")
-    exit(1)
+    exit(2)
```
Comment (jamc-code) [inline country_codes.py:57]: the different exit codes are so we can handle errors properly in the main script

File: vpn_module.sh [hunk 3]
```diff
@@ -119,61 +131,63 @@ vpn_location_menu() {
...
 
+		# TODO(julia) can these be quoted? i feel like last time it broke something
```
Comment (shervinsahba) [inline vpn_module.sh]: The `VPN_CODES[0]` through `[9]` break if they have quotes for the stock Mullvad setup. Originally, they are passing a country and city code as two arguments to `mullvad relay set location`, so they need to be globbed for that behavior. 

The remaining ones are country codes only, so should be fine in quotes.

The `$rofi_location` may not work with quotes either, for similar reasons, if I remember right. 

Thanks for all these contributions Julia! It's a busy week for me, but I'll test it out on my end soon. These look great.

Comment (jamc-code) [reply vpn_module.sh]: ahhhhh that's what it was! thanks for the explanation, i wasn't really getting it. also, no problem! i'm happy to help out, it's a super useful module.

**Comment by jamc-code** [comment]
Hey! I cleaned up the integration of the country code converter and updated the README to reflect that logging issue you found the other week. Hope you're doing well!
\- Julia

**Pull request merged**